src/components/form-type/field-checkbox-type/field-checkbox.component.tsx

```diff
@@ -2,69 +2,61 @@ import React, { memo, FC } from 'react';
 import { compose } from 'redux';
 import type { WrappedFieldProps } from 'redux-form';
 
-const types = [
-  {
-    id: 1,
+import {
+  withTranslations,
+  Props as TranslationsProps
+} from '../../../providers/translations';
 
-    label: 'Data'
-  },
-  {
-    id: 2,
-    label: 'Kodelister'
-  },
-  {
-    id: 3,
-    label: 'Tesauri'
-  },
-  {
-    id: 4,
-    label: 'Taksonomi'
-  },
-  {
-    id: 5,
-    label: 'Testdata'
-  }
-];
+interface ExternalProps {
+  types?: any[];
```

Comment:
Hvorfor `any` her? Vet vi objektstrukturen på forhånd? I så fall kan den kanskje types her?

Reply:
var any som ble brukt for alt lignende i repoet, fortsatte bare i samme stil. Men kan jo legge til